docs/application-development/foundations.md

```diff
@@ -720,12 +720,16 @@ If needed, you can customize `TEMPORAL_TASK_QUEUE`; the following example defaul
 ```typescript
 export function getEnv(): Env {
   return {
-    address: "web.<Namespace_ID>.tmprl.cloud", // NOT web.foo.bar.tmprl.cloud
-    namespace: "your.namespace", // as assigned
-    clientCertPath: "foobar.pem", // in project root
-    clientKeyPath: "foobar.key", // in project root
-    taskQueue: process.env.TEMPORAL_TASK_QUEUE || "hello-world-mtls", // just to ensure task queue is same on client and worker, totally optional
-    // // not usually needed
+    // NOT web.foo.bar.tmprl.cloud
```

Comment:
This comment says not, but the code below shows `web.foo.bar...`

Maybe a brief explanation "don't get confused by the Temporal Web UI address and your Temporal Cloud connection address - or something
Took me a hot minute to figure out wtf this comment was about -

Reply:
Hmm I see...
Well - make a note to come back and fix that comment at some point because it is confusing af.